bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ResponseQueueReader.java

```diff
@@ -58,11 +59,19 @@ private static synchronized Timer getFirstResponseTimer() {
   private ClientCallStreamObserver<ReadRowsRequest> requestStream;
   private AtomicInteger markerCounter = new AtomicInteger();
 
+  private static final long DEFAULT_WAIT_FOR_ROWS_TIMEOUT = TimeUnit.MINUTES.toMillis(10);
```

Comment:
Can this share the same value as the watchdog?

Reply:
Done =)